README.md

```diff
@@ -79,13 +78,12 @@ jobs:
       - name: Run the MC client
         uses: headlesshq/mc-runtime-test@3.0.0
         with:
-          mc: 3.0.0
+          mc: 1.21.4
```

Comment:
`mc: 1.21.4 # x-release-please-version`

Reply:
hmmmmmm... yeah Ill think about it later

Reply:
I still really dislike this because it looks like a comment, which could confuse readers.